avro-converter/src/test/java/io/confluent/connect/avro/AvroDataTest.java

```diff
@@ -1733,4 +1859,95 @@ private void assertNonNullSchemaValue(SchemaAndValue schemaAndValue) {
   }
 
 
+  protected void assertSchemaEquals(
```

Comment:
Because the Avro schema's default values are not always the same type as what was provided to the Avro SchemaBuilder. This is likely a bug within Avro itself.